Sources/QuickTests/Helpers/QuickSpec+QuickSpec_MethodList.m

```diff
@@ -0,0 +1,23 @@
+#import "QuickSpec+QuickSpec_MethodList.h"
+#import <objc/runtime.h>
+
+
+@implementation QuickSpec (QuickSpec_MethodList)
+
++ (NSSet<NSString*> *)allSelectors {
+    id t = [[[self class] alloc] init];
+    NSMutableSet<NSString*> *allSelectors = [NSMutableSet set];
+    
+    unsigned int methodCount = 0;
+    Method * mlist = class_copyMethodList(object_getClass(t), &methodCount);
+    NSLog(@"%d methods", methodCount);
```

Comment:
Please remove this `NSLog`, and the one below. If the information is relevant, we should have an automated test for it--which we do, in this case! :+1: